api/HTMLMediaElement.json

```diff
@@ -3538,10 +3538,10 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "6"
+              "version_added": "3.2"
```

Comment:
That the `src` attribute should have shipped one release after the rest of the API seems fishy, but not impossible.

https://trac.webkit.org/changeset/27277/webkit shows it all landing at once, but that was at WebKit trunk 523.11.0, so I think the Safari support was released from a branch first and the WebKit source history shouldn't be trusted here.

Still, how confident are about the distinctions between Safari 3.1 and 3.1 in this PR? Were both tested, and what's the UA string of both?

Reply:
Both were indeed tested, yeah.  Admittedly, due to internet connection issues in the Mac-based VMs for Safari 3.x, I'm not able to _actually_ submit reports to the collector; in other words, they were manually written up, though I used the mdn-bcd-collector to see the results.  I'm testing in Safari 3.1.2 and 3.2.3, however (which are the latest patch versions for each, as per [Wikipedia](https://en.wikipedia.org/wiki/Safari_version_history#Safari_3)).

Reply:
Alright, then I suppose they might have forgotten to add the reflected `src` attribute initially, which is an easy enough mistake to make, and then fixed it before anything was merged into WebKit.